docker/ports.conf.gen

```diff
@@ -1,6 +1,7 @@
 Listen {%DECK_HOST%}:{%DECK_PORT%}
 
 <IfModule ssl_module>
+        SSLPassPhraseDialog exec:/etc/apache2/passphrase
```

Comment:
It writes an empty line which is interpreted as "no password". I've verified this in the VM world, it's probably worthwhile that I check it works here too.